src/component/Registration/SignIn.css

```diff
@@ -0,0 +1,66 @@
+.registrationForm {
+    background-color: #fff;
+    color: black;
+    width: 400px;
+    border-radius: 8px;
+    padding: 8px 31px; 
```

Comment:
I'd recommend you use `rem` instead of `px`

Reply:
changed to rem already